NitroxClient/MonoBehaviours/Gui/MainMenu/MainMenuMultiplayerPanel.cs

```diff
@@ -132,37 +130,37 @@ public static async Task OpenJoinServerMenuAsync(string serverIp, string serverP
 
         private void ShowAddServerWindow()
         {
-            serverNameInput = "local";
-            serverHostInput = "127.0.0.1";
-            serverPortInput = ServerList.DEFAULT_PORT.ToString();
-            showingAddServer = true;
-            shouldFocus = true;
+            IEnumerator SetWindowComponents()
+            {
+                serverNameInput = "local";
+                serverHostInput = "127.0.0.1";
+                serverPortInput = ServerList.DEFAULT_PORT.ToString();
+                showingAddServer = true;
+                shouldFocus = true;
+                yield return null;
+                uGUI_MainMenu.main.canvasGroup.interactable = false;
+            }
+            CoroutineHost.StartCoroutine(SetWindowComponents());
```

Comment:
This works but it's not necessary to wait until next frame to disable input. Conceptually I prefer it like this, where the input is disabled as soon as possible.

```cs
            uGUI_MainMenu.main.canvasGroup.interactable = false;
            serverNameInput = "local";
            serverHostInput = "127.0.0.1";
            serverPortInput = ServerList.DEFAULT_PORT.ToString();
            showingAddServer = true;
            shouldFocus = true;
```

Reply:
Understood, making the edits now, thank you